app/src/main/java/com/example/themeal/data/source/api/MealService.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.themeal.data.source.api
+
+import com.example.themeal.data.model.DataResponse
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface MealService {
+
+    @GET("search.php")
+    fun getMealByName(@Query("s") name: String): Call<DataResponse.MealResponse>
```

Comment:
```suggestion
    suspend fun getMealByName(@Query("s") name: String): DataResponse.MealResponse
```
Hơn nữa ở đây e có thể trả về luôn kiểu data và bắt exception ở chỗ khác , và e đã có xử lí bất đồng bộ trên viewmodel rồi vậy nên ko cần enqueue ở đay nữa